Form/Type/MediaType.php

```diff
@@ -77,14 +76,34 @@ public function buildView(FormView $view, FormInterface $form, array $options)
 
     /**
      * {@inheritdoc}
+     *
+     * @deprecated To be removed when bumping to Symfony 2.7+, use MediaType::configureOptions() instead
      */
     public function setDefaultOptions(OptionsResolverInterface $resolver)
     {
+        if ($resolver instanceof OptionsResolver) {
```

Comment:
`setDefaultOptions` or `configureOptions` are called by `ResolvedFormType` so you don't have to care of this.

BTW, the deprecated notice is called on this class: https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Form/ResolvedFormType.php#L214-L216

Please follow classic method like is done on many bundle like this one: https://github.com/FriendsOfSymfony/FOSUserBundle/pull/1821

This works perfectly and is even more readable and simpler.

Regards